plugins/inventory/cobbler.py

```diff
@@ -56,6 +56,15 @@
         default: []
         elements: str
         version_added: 4.4.0
+      inventory_hostname:
+        description:
+          - What to use for the ansible inventory hostname.
+          - By default the networking hostname is used if defined, otherwise the dns name of the management or first non-static interface.
```

Comment:
```suggestion
          - By default the networking hostname is used if defined, otherwise the DNS name of the management or first non-static interface.
```